src/views/SettingsPage.vue

```diff
@@ -43,7 +43,7 @@
       </ion-item>
       <ion-item>
         <ion-label>
-          Enable personal feed
+          Enable personal data on Explore
```

Comment:
Enable personal feed on Explore.

Reply:
> Enable personal feed on Explore

I would have used this, but it also uses personal data for the "Projects Friends Loved" section.